src/qt/sendcoinsdialog.cpp

```diff
@@ -628,7 +628,7 @@ void SendCoinsDialog::updateSmartFeeLabel()
     if(!model || !model->getOptionsModel())
         return;
 
-    int nBlocksToConfirm = (int)25 - (int)std::max(0, std::min(24, ui->sliderSmartFee->value()));
+    int nBlocksToConfirm = (int)defaultConfirmTarget - (int)std::max(ui->sliderSmartFee->minimum(), std::min(ui->sliderSmartFee->maximum(), ui->sliderSmartFee->value()));
```

Comment:
[quote]
When setting this property, the minimum is adjusted if necessary to ensure that the range remains valid. Also the slider's current value is adjusted to be within the new range.
[/quote]

Seems like this IS the case...